src/views/home/components/header/header.view.tsx

```diff
@@ -0,0 +1,32 @@
+import { Link } from "react-router-dom";
+
+import { ReactComponent as Logo } from "src/assets/hermez-logo.svg";
+import { ReactComponent as Settings } from "src/assets/icons/settings.svg";
```

Comment:
When you import an icon it would be great to add the `Icon` prefix:

```
import { ReactComponent as SettingsIcon } from "src/assets/icons/settings.svg";
import { ReactComponent as QRIcon } from "src/assets/icons/qr.svg";
```